packages/manager/modules/pci/src/projects/project/instances/instance/instance.html

```diff
@@ -96,6 +98,7 @@
                 <oui-tile-button
                     data-on-click="$ctrl.softRebootInstance()"
                     data-ng-if="$ctrl.instance.statusGroup === 'ACTIVE'"
+                    data-disabled="$ctrl.instance.isShelvingOrUnshelving()"
```

Comment:
Same here, if `statusGroup` is `ACTIVE`, `status` will be in `['ACTIVE', 'RESCUED', 'RESIZED']` and can't be at the same time in `SHELVING/UNSHELVING`

The conditions will never be ok.

Reply:
Thanks for this observation.
Right the both `if` and `disabled` are no compliance.
I removed the disabled one.